scripts/actions/abilities/gauge.lua

```diff
@@ -8,27 +8,11 @@
 local abilityObject = {}
 
 abilityObject.onAbilityCheck = function(player, target, ability)
-    if player:getPet() ~= nil then
-        return xi.msg.basic.ALREADY_HAS_A_PET, 0
-    end
-
-    return 0, 0
+    return xi.job_utils.beastmaster.onAbilityCheckGuage(player, target, ability)
```

Comment:
Gauge is typoed here and in the job_utils lua

Reply:
least i was consistent ;)

Reply:
Yup haha